synapse/lib/nexus.py

```diff
@@ -134,6 +142,11 @@ async def __anit__(self, cell):
         self.nexslog.setIndex(maxindx)
 
         async def fini():
+            for wind in self._linkmirrors:
+                await wind.fini()
+
+            for dist in self._mirrors:
+                await dist.fini()
```

Comment:
Was this an existing bug?

Reply:
I think it would usually end up fini'ing itself anyways but it seems better to do it explicitly here.